steward/src/commands/keys/eth/import.rs

```diff
@@ -1,13 +1,13 @@
 use super::show::ShowKeyCmd;
 use crate::application::APP;
 use abscissa_core::{clap::Parser, Application, Command, Runnable};
-use k256::pkcs8::ToPrivateKey;
+use k256::{pkcs8::ToPrivateKey, SecretKey};
 use signatory::FsKeyStore;
 use std::path;
 
 #[derive(Command, Debug, Default, Parser)]
 #[clap(
-    long_about = "DESCRIPTION \n\n Import an external Eth key.\n This command will recover a Eth key, storing it in the keystore. \n It takes a keyname and bip39-mnemonic."
+    long_about = "DESCRIPTION \n\n Import an external Eth key.\n This command will recover a Eth key, storing it in the keystore. \n It takes a keyname and Eth private key."
```

Comment:
This should be updated to specify it takes either a private key or a mnemonic, and they they are mutually exclusive.